src/topics/attester/attester.test.ts

```diff
@@ -0,0 +1,170 @@
+import { BigNumber, ethers } from "ethers";
+import hydraS1Simple from "../../../attesters/hydra-s1-simple";
+import resetTestInfrastructure from "../../infrastructure/test-infrastructure";
+import { AttestationsCollection } from "../attestations-collection";
+import { Badge } from "../badge";
+import { Attester } from "./attester";
+import {
+  AllConstructedAttesters,
+  getConstructedAttesters,
+  getConstructedNetworkAttesters,
+} from "./attester.helper";
+import { AttesterNetwork } from "./types";
+
+jest.mock("../../../attesters/hydra-s1-simple", () => {
```

Comment:
You should not use any mock here. I will show you an example with  group-generators (in another PR)